### ISSUE TITLE
Proceedings Missing: Field Matters Workshop @ EACL'23 

### ISSUE THREAD

**Issue by oserikov** [acl-org/acl-anthology#2583]
### Confirm that this is a bug report

- [X] I want to report an issue that does _not_ concern paper or author metadata.
- [X] I have searched for similar existing issues first.

### Problem Description

Dear team of aclanthology. I'm the organizer of the [field matters workshop](https://field-matters.github.io) held this year at EACL'23. The proceedings of our workshop weren't published yet, and over the communication with the EACL workshops chairs we hypothesise that there is a huge publication process going on on the aclanthology. 

I'm reaching out here wondering if I personally can contribute my skills and make our proceedings published, finally. **It's 1.5 months past the conference and our participants may face dramatical consequences** of not being able to formally report the acceptance to the conference workshop.

**Comment by akoehn** [comment]
The relevant pull requests for eacl and its workshops are:
 - https://github.com/acl-org/acl-anthology/pull/2495/files
 - https://github.com/acl-org/acl-anthology/pull/2499/files
 - https://github.com/acl-org/acl-anthology/pull/2508/files
 - https://github.com/acl-org/acl-anthology/pull/2523

the field matters workshop is not listed in any of these, which leads me to believe that the data was simply not handed over to the anthology.

Regarding your hypothesis: The publication process is actually quite straight-forward and transparent:
 - publication chairs / workshop organizers do a data export (e.g. ACLPUB) and send data & papers to us
 - we create the XML files as a pull request in this repository and upload the PDFs to the server
 - once the pull request is merged, the anthology is recompiled automatically to contain the ws/event

Probably you and/or @anthology-assist can say at what stage the field matters workshop is currently hanging.  It might be that the first step is not even done yet for that workshop. Generally speaking, there is usually no "invisible" delay on the anthology side.

**Comment by oserikov** [comment]
Dear @zeeraktalat ! This thread is regarding the Field Matters proceedings. We had this conversatio over email previously. Could you help me to understand what is happening? 

What I know are these pieces:
* there exists this repo since May, 2 [eacl-2023/Field-Matters](https://github.com/eacl-2023/Field-Matters), 
* we also had this email thread late April, 13 . It is then when @EkaterinaVoloshina reported to me for the first time that the proceedings are submitted
* Then we've also had the post-conference iteration with the proposed fixes in the authors list

It seems to me that we either misunderstand your directives on the FM side, or there's something going on the EACL/ACL side I have no knowledge of. I'm eager to contribute all my expertise, skills and time to make proceedings happen.

CC: @ivri

**Comment by anthology-assist** [comment]
Hi @oserikov , thank you for submitting the bug report; and @akoehn thank you for the  detailed explanation on the ingestion procedure.

During the EACL 2023 ingestion process, at first FM was put on hold awaiting for more information; then several workshops had issues with their ingestion material, FM being one of them.  There has been back and forth emailing and fixing regarding different problems going on with different workshops, FM simply fell through the crack. 

It has been ingested, and should be merged and live on anthology soon.
